packages/neth/src/lib/neth-lib.ts

```diff
@@ -104,8 +104,9 @@ export const initConnection = ({
   keyStore = new BrowserLocalStorageKeyStore();
   near = new Near({
     ...network,
-    deps: { keyStore },
+    keyStore,
```

Comment:
Can you please keep just this change and revert other changes?

I am sorry if my comment on the other PR was confusing, at the moment we have been just testing wallet-selector with **NAJ v1.1.0** but we did not decide if we should update the version yet.

I checked and this change works fine with `v0.44.2` too.